ci/travis/install-dependencies.sh

```diff
@@ -53,7 +53,7 @@ if [[ "$PYTHON" == "3.6" ]] && [[ "$platform" == "linux" ]]; then
     opencv-python-headless pyyaml pandas==0.24.2 requests \
     feather-format lxml openpyxl xlrd py-spy pytest-timeout networkx tabulate aiohttp \
     uvicorn dataclasses pygments werkzeug kubernetes flask grpcio pytest-sugar pytest-rerunfailures pytest-asyncio \
-    blist scikit-learn numba
+    blist scikit-learn numba parameterized pytest
```

Comment:
pytest should already be installed

Reply:
I was a bit surprised that you install pytest as part of ray and not introduce ray[test]

Reply:
maybe let's do this in a separate PR, since I think it'll affect some documentation